tests/python_on_whales/components/buildx/imagetools/test_imagetools_cli_wrapper.py

```diff
@@ -68,3 +72,52 @@ def test_imagetools_append(docker_registry):
     docker.buildx.imagetools.create([base_image_busybox], tags=[new_image])
     docker.buildx.imagetools.create([base_image_alpine], tags=[new_image], append=True)
     docker.pull(new_image)
+
+
+def test_imagetools_create_annotations_type_validation():
+    with pytest.raises(TypeError) as err:
+        docker.buildx.imagetools.create(
+            ["python:3.13.0"], annotations="not-a-dict", dry_run=True
+        )
+    assert "must be a dict" in str(err.value)
+
+
+@patch("python_on_whales.components.buildx.imagetools.cli_wrapper.run")
+def test_imagetools_create_with_annotations_command_construction(mock_run):
```

Comment:
Would it be possible in addition of this patched test to have a test running the real command? To make sure the inputs and outputs are what is expected by the real buildkit engine?